### DISCUSSION TITLE
Stalebot is pissing me off

### DISCUSSION THREAD

**Discussion by TiagoTiago** [oobabooga/text-generation-webui#6152] [Ideas]
Can we please shutdown this stupid thing? Sweeping unsolved bugs and ignored  feature requests under the rug makes no fucking sense. Marking "stale" issues as "Completed" is idiotic.

**Comment by Kwisss** [comment]
Stalebot marks the dead of a repository 10/10 times,
the problem is, I don't know yet what the next best thing is.....
